flashinfer/fused_moe/cute_dsl/blackwell_sm12x/_moe_dynamic/gated.py

```
@@ -0,0 +1,5076 @@
+        expert_tile_base: cute.Tensor,
+        input_global_scale: cute.Tensor,
+        alpha: cute.Tensor,
+        down_alpha: cute.Tensor,
+        global_scale: cute.Tensor,
+        scatter_output: cute.Tensor,
+        token_map: cute.Tensor,
+        token_weights: cute.Tensor,
+    ):
+        """Kernel entry point."""
+        tidx, _, _ = cute.arch.thread_idx()
+        bidx, _, bidz = cute.arch.block_idx()
```

Ruff (0.16.0)

[warning] Unpacked variable `bidx` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)